Diagnostics/HNS_diagnostics.md

```diff
@@ -0,0 +1,35 @@
+# Container Networking Diagnostics
+
+> If you face DNS or networking issues with containers, **the best solution** is to ensure you are running the _**most recent Windows release**_. We worked  very hard to iron out container networking issues in Windows 1803, so we encourage you to update and benefit from the smoother experience provided by the newest releases.
```

Comment:
Similar to changes I made here: https://hackmd.io/hpKY9Pn_TSKRO_3WWyEjdQ?both#  please ensure we call out this is for Windows Server, version XXXX and Windows 10 Creators Update